dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -179,23 +172,37 @@ HRESULT CGuidItemExtractIcon_CreateInstance(LPCITEMIDLIST pidl, REFIID iid, LPVO
         }
     }
 
-    if (HCR_GetIconW(xriid, wTemp, iconname, MAX_PATH, &icon_idx))
+    /* Prepare registry path for loading icons of My Computer and other shell extensions */
+    WCHAR xriid[MAX_PATH];
+
+    swprintf(xriid, L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\CLSID\\{%08lx-%04x-%04x-%02x%02x-%02x%02x%02x%02x%02x%02x}",
```

Comment:
There's a better way to convert a GUID to a string. `StringFromGUID2` and a helper function etc.

Reply:
Switched to `StringFromGUID2` with a helper function.